api/hasura/actions/_handlers/allocationCsv.test.ts

```diff
@@ -60,33 +61,67 @@ function getMockCircleDistribution(type?: DistributionType | undefined) {
     case DISTRIBUTION_TYPE.FIXED:
       distributions.push({
         distribution_type: DISTRIBUTION_TYPE.FIXED,
+        distribution_json: {
```

Comment:
Sorry to be a stick in the mud about this, but an easy way to track that this mock matches the query is to type this function to return a piece of the actual query. So if the query is ever updated, this will fail to typecheck until it's updated as well and will be more maintainable.

Reply:
great suggestion have updated it